packages/wonder-blocks-modal/components/modal-panel.js

```diff
@@ -85,16 +81,22 @@ export default class ModalPanel extends React.Component<Props> {
                     //    mediaSize === "small" && styles.smallCloseButton,
                 ]}
             >
-                <IconButton
-                    icon={icons.dismiss}
-                    // TODO(mdr): Translate this string for i18n.
-                    aria-label="Close modal"
-                    onClick={onClickCloseButton}
-                    kind={
-                        topBackgroundColor === "dark" ? "primary" : "tertiary"
-                    }
-                    light={topBackgroundColor === "dark"}
-                />
+                <Context.Consumer>
+                    {({closeModal}) => (
+                        <IconButton
+                            icon={icons.dismiss}
+                            // TODO(mdr): Translate this string for i18n.
+                            aria-label="Close modal"
+                            onClick={closeModal || onClose}
```

Comment:
I wonder if this should have similar behavior to the old version: if both callbacks are provided, maybe they should both be called?

Reply:
I can give that a try and see what happens. I'm curious to see if any of the callback get double called.

Reply:
I thought about it some more and I think we should disallow using both.  I can't think of a situation where we'd want to be able to close a modal using the backdrop, default close button, and possible a custom close button but do something special for only the default close button.  I added a `console.warn` if both `closeModal` and `onClose` are set to help people do the right thing in that situation.